lib/walkman/tape.ex

```diff
@@ -49,10 +54,18 @@ defmodule Walkman.Tape do
     preserve_order = Keyword.get(options, :preserve_order, true)
     global = Keyword.get(options, :global, Application.get_env(:walkman, :global, false))
 
+    module_changes =
+      Keyword.get(options, :module_changes, Application.get_env(:walkman, :module_changes))
```

Comment:
Does this work when you haven't set it?

Reply:
I notice that this is missing the default value (like on line 55)

Reply:
I also see that there is no test for the default.

Reply:
I have fixed it. I thought the default would be set because of the default set at the struct definition but it didn't work as I expected.